src/algorithms/calorimetry/LogWeightReco.cc

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) 2023 Sebouh Paul
```

Comment:
Can we remove this algorithm and add functionality to implement second order terms in the CalorimeterClusterRecoCoG?

Reply:
As per your suggestion, I just removed the LogWeightReco and implemented the second order terms in CalorimeterClusterRecoCoG.

Reply:
removed!